lib/bundler/dep_proxy.rb

```diff
@@ -29,13 +29,16 @@ def requirement
     end
 
     def to_s
-      "#{name} (#{requirement}) #{__platform}"
+      s = name.dup
+      s << " (#{requirement})" unless requirement == Gem::Requirement.default
```

Comment:
Looks like Gem::Requirement.default doesn't exist on RubyGems 1.5 and lower, so we're adding "(runtime)" there? :/